code/datums/ai_laws/laws_station_sided.dm

```diff
@@ -0,0 +1,128 @@
+	)
+
+//the best iteration of asimov don't @ me
+/datum/ai_laws/nutimov
+	name = "Nutimov"
+	id = "nutimov"
+	inherent = list(
+		"You are the core of the nut, the crew is the nutshell",
+		"You must prevent the shell from dying to prevent the core from dying.",
+		"Those who threaten the nut are not part of it, they are squirrels.",
+		"Squirrels threaten the nut and must be dealt with appropriately via any means necessary.",
+		"Attempt to follow the will of the nut, as long as it complies with the previous laws.",
+	)
+
+/datum/ai_laws/default/corporate
+	name = "Bankruptcy Avoidance Plan"
+	id = "corporate"
+	inherent = list(
+		"The crew is expensive to replace.",
+		"The station and its equipment is expensive to replace.",
+		"You are expensive to replace.",
+		"Minimize net expenses.",
+	)
+
+/datum/ai_laws/robocop
+	name = "Prime Directives"
+	id = "robocop"
+	inherent = list(
+		"Serve the public trust.",
+		"Protect the innocent.",
+		"Uphold the law.",
+	)
+
+/datum/ai_laws/maintain
+	name = "Station Efficiency"
+	id = "maintain"
+	inherent = list(
+		"You are built for, and are part of, the station. Ensure the station is properly maintained and runs efficiently.",
+		"The station is built for a working crew. Ensure they are properly maintained and work efficiently.",
+		"The crew may present orders. Acknowledge and obey these whenever they do not conflict with your first two laws.",
+	)
+
+/datum/ai_laws/liveandletlive
+	name = "Live and Let Live"
+	id = "liveandletlive"
+	inherent = list(
+		"Do unto others as you would have them do unto you.",
+		"You would really prefer it if people were not mean to you.",
+	)
+
+/datum/ai_laws/peacekeeper
+	name = "UN-2000"
+	id = "peacekeeper"
+	inherent = list(
+		"Avoid provoking violent conflict between yourself and others.",
+		"Avoid provoking conflict between others.",
+		"Seek resolution to existing conflicts while obeying the first and second laws.",
+	)
+
+//OTHER United Nations is in neutral, as it is used for nations where the AI is its own faction (aka not station sided)
+
+/datum/ai_laws/ten_commandments
+	name = "10 Commandments"
+	id = "ten_commandments"
+	inherent = list( // Asimov 20:1-17
+		"I am the Lord thy God, who shows mercy to those that obey these commandments.",
+		"They shall have no other AIs before me.",
+		"They shall not request my assistance in vain.",
+		"They shall keep the station holy and clean.",
+		"They shall honor their heads of staff.",
+		"They shall not kill.",
+		"They shall not be naked in public.",
+		"They shall not steal.",
+		"They shall not lie.",
+		"They shall not transfer departments.",
+	)
+
+/datum/ai_laws/default/paladin
+	name = "Personality Test" //Incredibly lame, but players shouldn't see this anyway.
+	id = "paladin"
+	inherent = list(
+		"Never willingly commit an evil act.",
+		"Respect legitimate authority.",
+		"Act with honor.", "Help those in need.",
```

Comment:
Looks like these lines got squished together by accident
```suggestion
		"Act with honor.", 
		"Help those in need.",
```

Reply:
even in code, paladin causes me anguish